### PR TITLE
Override label, show text to copy

### PR DESCRIPTION
**Pull request by aisurfer**
How it looks like
Before click
image

After click (both text and button are clickable)
image

### CODE REVIEW

File: st_copy_to_clipboard/__init__.py [hunk 1]
```diff
@@ -32,6 +50,10 @@ def main():
     st.write("## Example")
     text = st.text_input("Enter text to copy to clipboard", value="Hello World")
     st_copy_to_clipboard(text)
+    st_copy_to_clipboard(text, before_copy_label='📋Push to copy', after_copy_label='✅Text copied!')
+    st_copy_to_clipboard(text, before_copy_label='Push to copy', after_copy_label='Text copied!', show_text=True)
```
Comment (aisurfer) [inline st_copy_to_clipboard/__init__.py:54]: Hi @mmz-001 !
Thank you for the awesome component.
I make two modifications that might be useful for others. It just works.
Would be great to push it to main and to make it available it pip package

**Comment by shayel8** [comment]
great idea! hope it will be merged soon :)

**Comment by yourfavtheo** [comment]
Great PR ! Definitely what I was looking for 👍🏾 
Can you please merge it @mmz-001 ?

**Review by mmz-001 — APPROVED** [review]
lgtm🚀

**Pull request merged**